Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1072,10 +1072,43 @@ std::variant<bool, CLuaMultiReturn<float, float, float, float>> CLuaPedDefs::Get
     return std::make_tuple(x, y, z, w);
 }
 
-bool CLuaPedDefs::SetElementBoneMatrix(lua_State* const luaVM, CClientPed* entity, std::uint32_t boneId, CMatrix boneMatrix)
+bool CLuaPedDefs::SetElementBoneMatrix(lua_State* const luaVM)
```

Comment:
That wont work for argument parser.

Reply:
In this case it would be through `LUA_DECLARE`, correct? I've made the change, if that's all it is, I'll send.

Reply:
I decided to follow the declaration pattern of other functions. I've sent the changes.

Reply:
Did you check if it works as intended?

Reply:
Yes, I never send a commit without testing it.

Reply:
Yes, I never send a commit without testing it.

Reply:
Yes, I did. All the parameters were working and the function worked correctly. I only changed it because you told me to, but it was already working.